test_data/yast/xfs/xfs_partition_opensuse.yaml

```diff
@@ -0,0 +1,24 @@
+---
+disks:
+  - name: vda
+    table_type: gpt
+    allowed_unpartitioned: 0.00GB
+    partitions:
+      - name: vda2
+        role: operating-system
+        partition_type: primary
```

Comment:
could you please delete the partition_type key in the file? that is used just for msdos. `should_mount` is still needed in `validate_read_write`.